docs/index.html

```diff
@@ -19,6 +19,108 @@
 		nav.app-nav li ul {
 			min-width: 100px;
 		}
+
+		/* CarbonAds starts here */
+		#carbonads {
+			font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+				Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Helvetica, Arial,
+				sans-serif;
+		}
+
+		#carbonads {
+			display: flex;
+			position: fixed;
+			bottom: 10px;
+			right: 10px;
+			max-width: 150px;
+			background-color: hsl(0, 0%, 98%);
+			box-shadow: 0 1px 4px 1px hsla(0, 0%, 0%, 0.1);
+			z-index: 100;
+		}
+
+		#carbonads a {
+			color: inherit;
+			text-decoration: none;
+		}
+
+		#carbonads a:hover {
+			color: inherit;
+		}
+
+		#carbonads span {
+			position: relative;
+			display: block;
+			overflow: hidden;
+		}
+
+		#carbonads .carbon-wrap {
+			display: flex;
+			flex-direction: column;
+		}
+
+		.carbon-img {
+			display: block;
+			margin: 0;
+			line-height: 1;
+		}
+
+		.carbon-img img {
+			display: block;
+			max-width: 100% !important;
+		}
+
+		.carbon-text {
+			font-size: 13px;
+			padding: 10px;
+			line-height: 1.5;
+			text-align: left;
+		}
+
+		.carbon-poweredby {
+			display: block;
+			padding: 8px 10px;
+			background: repeating-linear-gradient(-45deg,
+				transparent,
+				transparent 5px,
+				hsla(0, 0%, 0%, 0.025) 5px,
+				hsla(0, 0%, 0%, 0.025) 10px) hsla(203, 11%, 95%, 0.4);
+			text-align: center;
+			text-transform: uppercase;
+			letter-spacing: 0.5px;
+			font-weight: 600;
+			font-size: 9px;
+			line-height: 1;
+		}
+
+		@media only screen and (min-width: 320px) and (max-width: 759px) {
+			#carbonads {
+				float: none;
+				margin: 20px 0;
+				max-width: 330px;
+			}
+
+			#carbonads .carbon-wrap {
+				display: flex;
+				flex-direction: row;
+			}
+
+			.carbon-img {
+				display: block;
+				margin-bottom: 8px;
+				max-width: 130px;
+				line-height: 1;
+			}
+
+			.carbon-img img {
+				display: block;
+				margin: 0 auto;
+				max-width: 130px;
+				width: 130px;
+				height: auto;
+			}
+		}
```

Comment:
Can we create it as a css file? for use in both locations.

Reply:
I tried, for some reason it's not loading that file.

Reply:
well, I try it.

Reply:
there are few [here](https://codepen.io/collection/AyEoKk/?__cf_chl_jschl_tk__=90a2969943902982e6f4af8d84f0c9ffc2831e7b-1597560043-0-AfrHJDIz9I3eXlHh5jILZCelQdj_BRK98h8PDoXhnfW9IoNyglegU9XdfIliC_pFReugMXn7O3A85VL258ETHodRBcms_ZpHf1-mEBzvDco3oSopfcnUFV4tohgVkpF3GjU3zAecDp_VIUElfnCTyBrWYysT0ZePyiFj_jK1C9uzjVKiGKBgTCVE6X83_66hUNKFQ7F8kJdUeS8NbrFU0nSC2A8kCeE7GlZit7BYIKESXzVoSsePiACdlhOzzqTrkQ8jywsbZOEFOS93VoBJF2ZbS3nu3u4MBm6fzkyQNviIW3bXmRwadEchtgwP8E4g_N7dhsdAuj_30-1KLFyo0TB2HhAMd6MZJOciSRqr00PY&cursor=ZD0wJm89MSZwPTEmdj00NzE4MjY5) for reference.

Reply:
> I tried, for some reason it's not loading that file.

I tried it, and it's true...